docs/configuring-metabase/localization.md

```diff
@@ -6,6 +6,54 @@ redirect_from:
 
 # Languages and localization
 
+The **Localization** settings allow you to set global defaults for your Metabase instance. You can find **Localization** under **Admin settings** > **Settings**.
+
+## Instance language
+
+The instance language is the default language across the Metabase UI, system [emails](./email.md), [dashboard subscriptions](../dashboards/subscriptions.md), and [alerts](../questions/sharing/alerts.md). People can pick a different language from their own [account settings](../people-and-groups/account-settings.md).
+
+## Report timezone
+
+Use **report timezone** to set a default display time zone for dates and times in Metabase. This is a display setting only, so changing the **report timezone** won't affect the time zone of any data in your database.
+
+**Report timezone** doesn't apply to `timestamp without time zone` data types, including the output of [`convertTimezone`](../questions/query-builder/expressions/converttimezone.md) expressions.
```

Comment:
Can you elaborate on the implications of this inertia with an example? What about workarounds?